pages/integrations/_one_repo_multi_org_github.md.erb

```diff
@@ -0,0 +1 @@
+If you want to integrate the same repository to multiple organisations, you need to link each organisation to Github using different Buildkite user accounts. A single Buildkite user can only be linked to a single GitHub user, and that Buildkite user's linked GitHub account should have admin permissions on the intended GitHub organization. If that is not the case, another Buildkite user (with the proper permissions) should install the integration. Each Github App can only be installed once per Github org. 
```

Comment:
There are a few style points to fix:

- Use US English (Organizations).
- Be very clear about what is Buildkite and what is GitHub.
- Use company names like their branding (GitHub).
- Add some more of the why (my assumptions).

Perhaps:

```suggestion
If you want to integrate the same repository into multiple Buildkite organizations, you need to link each organization to GitHub using different Buildkite user accounts. You must use different user accounts because there's a one-to-one relationship between a Buildkite user and a GitHub user. The user needs admin permissions on the GitHub organization to link it to Buildkite. You can only install the Buildkite app for GitHub once per GitHub organization.
```

Reply:
Thanks @mbelton-buildkite ! This is great!